README.md

```diff
@@ -1,16 +1,204 @@
 # Documentation Guidelines
+The official https://appwrite.io/docs documentation source code.
 
 ## 🚀 Contributing
+1. Clone the repository
+2. Update, add, or fix current docs
+3. Once content is ready, raise a PR
 
 ## Before Proposing Additions
+Documentation is an integral part of Appwrite and follows the same philosophy of quick to get started, easy to grow. Before proposing additions to Appwrite's documentation, think about where these additions fit best.
+- The **Getting Started** section helps a developer create their first Appwrite project and make their first requests so they can begin exploring Appwrite. Avoid adding unnecessary information to this section so a developer's first-impressions remains quick and smooth.
+- The **Guides** section helps a developer get comfortable with individual Appwrite services. Guides are structured to walk a developer through features of an Appwrite service. The topics should be ordered by level of complexity, where complex topics are positioned later into a guide.
```

Comment:
FYI, Tessa, Torsten, Eldad, and I had a talk about this section recently, and we may work on breaking down these pages since they're getting to be a little long.

Reply:
I agree. Please loop me in on this type of conversation, I'd love to stay in the loop 💯